### DISCUSSION TITLE
I want to set the shortcut key for switching tabs of bufferline

### DISCUSSION THREAD

**Discussion by PILIHU2022** [LazyVim/LazyVim#1385] [Q&A] [answered]
I want to set the shortcut key for switching tabs of bufferline, but it is not available after I follow the keymap configuration on the official website. How should I configure it to set the shortcut key for switching? Please provide code example if possible

This is the code of the official website:
```
-- tabs
map("n", "<leader><tab>l", "<cmd>tablast<cr>", { desc = "Last Tab" })
map("n", "<leader><tab>f", "<cmd>tabfirst<cr>", { desc = "First Tab" })
map("n", "<leader><tab><tab>", "<cmd>tabnew<cr>", { desc = "New Tab" })
map("n", "<leader><tab>]", "<cmd>tabnext<cr>", { desc = "Next Tab" })
map("n", "<leader><tab>d", "<cmd>tabclose<cr>", { desc = "Close Tab" })
map("n", "<leader><tab>[", "<cmd>tabprevious<cr>", { desc = "Previous Tab" })
```

**Comment by abeldekat** [comment, marked as answer]
Hello @PILIHU2022,

Hoping I am interpreting your question correctly: I assume you want to be able to switch buffers.
The default keybindings are documented [here]( https://www.lazyvim.org/keymaps#general), search for the lines containing the word "buffer".

How to change a default general keymap is documented [here](https://www.lazyvim.org/configuration/general#keymaps)
As an example, overriding the default "previous buffer" keybinding:

`lua/config/keymaps.lua`
``` lua
vim.keymap.set("n", "<S-h>", function()
  vim.print("hello")
end, { desc = "Overriding Prev buffer" })
```

Both buffers and tabs(`:h tab-page`) are core `neovim` concepts. 
The UI of the bufferline plugin displays `neovim`'s buffers and tabs in *tab components*. 

Best regards!
